packages/web/src/scss/components/Checkbox/README.md

```diff
@@ -1,6 +1,6 @@
 # Checkbox
 
-## Basic usage:
+## Basic usage
```

Comment:
title case everywhere. Maybe we can lint this somehow too?

Reply:
Fixed.